content/influxdb/cloud/reference/internals/system-buckets.md

```diff
@@ -19,4 +19,87 @@ related:
   - /{{< latest "flux" >}}/stdlib/influxdata/influxdb/monitor/notify/
 ---
 
-{{< duplicate-oss >}}
\ No newline at end of file
+InfluxDB system buckets contain time series data used by and generated from the
+InfluxDB monitoring and alerting system and the task engine.
+
+Each InfluxDB organization includes the following system buckets:
+
+- [\_monitoring](#_monitoring-system-bucket)
+- [\_tasks](#_tasks-system-bucket)
+
+{{% note %}}
+The system bucket schemas below reference **InfluxDB data elements**.
+For more information, see [InfluxDB key concepts](/influxdb/cloud/reference/key-concepts/data-elements/).
+{{% /note %}}
+
+## \_monitoring system bucket
+The `_monitoring` system bucket stores InfluxDB data used to
+[monitor data and send alerts](/influxdb/cloud/monitor-alert/).
+
+**Data retention:** 7 days
+
+### \_monitoring bucket schema
+
+- **statuses** _(measurement)_
+    - **tags:**
+        - **\_check\_id:** check ID
+        - **\_check\_name:** check name
+        - **\_level:** level evaluated by the check (ok, info, warn, or crit)
+        - **\_source\_measurement:** original measurement queried by the check
+        - **\_type:** check type (threshold or deadman)
+        - _other tags inherited from queried data or added in the check configuration_
+    - **fields:**
+        - **\_message:** message generated by the check
+        - **\_source_timestamp:** original timestamp of the queried data
+        - _other fields inherited from queried data_
+- **notifications** _(measurement)_
+    - **tags:**
+        - **\_check\_id:** check ID that triggered the notification
+        - **\_check\_name:** check name that triggered the notification
+        - **\_level:** check-evaluated level that triggered the notification (ok, info, warn, or crit)
+        - **\_notification_endpoint_id:** notification endpoint ID
+        - **\_notification_endpoint_name:** notification endpoint name
+        - **\_notification_rule_id:** notification rule ID
+        - **\_notification_rule_name:** notification rule name
+        - **\_sent:** sent status (true or false)
+        - **\_source\_measurement:** original measurement queried by the check
+        - **\_type:** check type (threshold or deadman)
+        - _other tags inherited from queried data or added in the check configuration_
+    - **fields:**
+        - **\_message:** message generated by the check
+        - **\_source_timestamp:** original timestamp of the queried data
+        - **\_status_timestamp:** timestamp when the status (`_level`) was evaluated
+        - _other fields inherited from queried data_
+- **rejected_points** _(measurement)_
```

Comment:
@sanderson I couldn't make this work with `{{% cloud-only %}}` and the bulleted list; it broke everything that follows.

Reply:
Yeah, this is a known issue with this shortcode. I have an idea for how to fix it but I won't be able to do it until tomorrow. Right now, to use it in/with lists, you essentially have to do entire lists for both cloud and oss.